### ISSUE TITLE
Lua error on weapon replacement (item as replacement)

### ISSUE THREAD

**Issue by LibertyForce-Gmod** [LibertyForce-Gmod/Weapon-Properties-Editor#6]
[User report:](http://steamcommunity.com/workshop/filedetails/discussion/933160196/1291817208496568522/)

> I discovered that it is theoretically possible, by accidentally copypasting the entity version of the Armor Battery in the "will give players this weapon" box, which worked but spewed this console error

```lua
[Weapon Properties Editor] lua/autorun/lf_weapon_properties_editor.lua:232: attempt to call method 'GetMaxClip1' (a nil value)
1. v - lua/autorun/lf_weapon_properties_editor.lua:232
2. unknown - lua/includes/modules/hook.lua:84
```

**Comment by LibertyForce-Gmod** [comment]
Confirmed. The replacement can be any entity, not just a weapon.
Major oversight, needs a weapon check immediately.

**Comment by LibertyForce-Gmod** [comment]
Fixed with 02c6c495320ad460dc23269c22c6990ea9234dab
